packages/builder/builder-doc/en/config/tools/tsLoader.md

```diff
@@ -1 +1,87 @@
-TODO
+- Type： `Object` | `Function` | `undefined`
+- Default:`undefined`
+
+`ts-loader` is not enabled by default in the project. When `tools.tsLoader` is not undefined, it means that ts-loader is enabled, and babel-loader is disabled for TypeScript compilation.
```

Comment:
机翻的感觉有点重

it means that ts-loader is enabled, and babel-loader is disabled for TypeScript compilation.

->

Builder will use ts-loader instead of babel-loader to compile TypeScript code.

Reply:
我改下